forui/lib/src/widgets/select_group/select_group_controller.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/widgets.dart';
+
+/// A controller for a select group.
+abstract class FSelectGroupController<T> with ChangeNotifier {
+  final Set<T> _values;
+
+  /// Creates a [FSelectGroupController].
+  FSelectGroupController({Set<T> values = const {}}) : _values = {...values};
+
+  /// Handles a change in the selection.
+  // ignore: avoid_positional_boolean_parameters
+  void onChange(T value, bool selected);
```

Comment:
nitpick: could this method be renamed to `select`/`toggle`? It seems more fitting.